.github/ISSUE_TEMPLATE/5_docs_ticket.yaml

```
@@ -0,0 +1,46 @@
+name: 📝 Suggest edits or changes to docs
+description: Propose edits or enhancements to Prefect documentation
+labels: ["docs enhancement", "copy edit"]
```

We should use our existing label scheme 
```suggestion
labels: ["docs", "status:triage"]
```